interfaces/builtin/desktop.go

```diff
@@ -160,6 +160,63 @@ dbus (send)
 # Allow access to the ICC profiles in the home directory to
 # be referred to from colord
 owner @{HOME}/.local/share/icc r,
+
+
+# Allow to send updates to the desktop session about on going jobs
+# (for progress display in the task list)
+dbus (send)
+    bus=session
+    interface=com.canonical.Unity.LauncherEntry
+    member=Update
+    peer=(label=unconfined),
+
+# Allow to send updates to the desktop session about on going jobs
+# (for KDE Plasma specific details)
+dbus (send)
+    bus=session
+    interface=org.kde.JobViewServer*
+    path=/JobViewServer
+    member=requestView
+    peer=(label=unconfined),
+dbus (send)
+    bus=session
+    interface=org.kde.JobView*
+    path=/org/kde/notificationmanager/jobs/*
+    member={update,terminate}
```

Comment:
Without checking the details of the implementation in KDE, `terminate` does not match the description above. Is it really neccessary?

Reply:
yes, the name of this API isn't great IMO. But it's basically the `terminate` call which comes after the last `update` to say "OK, I'm done, you can forget about that job".